core/payment-driver/erc20/src/erc20/wallet.rs

```diff
@@ -141,27 +141,36 @@ pub async fn make_transfer(
     );
     let amount_big_dec = details.amount.clone();
     let amount = big_dec_to_u256(&amount_big_dec)?;
-    let gas_price = match gas_price {
-        Some(gas_price) => big_dec_gwei_to_u256(gas_price)?,
-        None => convert_float_gas_to_u256(get_polygon_starting_price()),
-    };
-    let max_gas_price = match max_gas_price {
-        Some(max_gas_price) => big_dec_gwei_to_u256(max_gas_price)?,
-        None => convert_float_gas_to_u256(get_polygon_maximum_price()),
+
+    let (gas_price, max_gas_price) = match network {
+        Network::Polygon => (
```

Comment:
Should this apply to Mumbai also?

Reply:
if possible i would double check with @scx1332 

i *think* it should apply to mumbai too

Reply:
On Mumbai you can duplicate Rinkeby behavior